### ISSUE TITLE
Use @Pattern directive to validate entries within a List/Array query argument

### ISSUE THREAD

**Issue by setchy** [graphql-java/graphql-java-extended-validation#18]
I was looking to use the @Pattern directive to validate that each String within an [String] query argument matches a given pattern (for example, 4 characters).

Is there a trick/workaround to getting this to work or is it a gap at the moment?

As an example, something like below would make sure that each [icao](https://en.wikipedia.org/wiki/ICAO_airport_code) provided is 4 characters in length
```
type Query {
    airports(
        icaos: [String] @Pattern(regexp: "^[a-zA-Z]{4}$")
    )
}
```

PS - this library is proving to be super helpful for validation conditions.  Great stuff!

**Comment by setchy** [comment]
Ended up create a pull request #19 that supports Lists

**Comment by setchy** [comment]
@bbakerman do you have a timeline now that the PR has been merged (along with bumping to v15) on when v0.0.5 will be released to maven central / jcenter etc
